spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/KubernetesClientInformerAutoConfiguration.java

```diff
@@ -57,6 +58,7 @@
 @ConditionalOnKubernetesDiscoveryEnabled
 @ConditionalOnBlockingOrReactiveEnabled
 @ConditionalOnCloudPlatform(CloudPlatform.KUBERNETES)
+@Conditional(ConditionalOnSelectiveNamespacesMissing.class)
```

Comment:
this is now annotated with `ConditionalOnSelectiveNamespacesMissing`, so it works as before. All integration tests pass, proving that everything that worked before is working now. I admit, that getting to this point where this works the way I expected it to, meant to create all other PRs that fixed various other things, cause I have this idea and PR for over a month...

Reply:
Thanks for all the hard work!